### DISCUSSION TITLE
Windows 工具列太無聊？讓貓貓來療育你吧！

### DISCUSSION THREAD

**Discussion by awkward-willy** [1101-GS4719-C/Python-Community-GS4719-C#608] [優秀專案]
前些日子看到朋友的工具列上有一隻貓正忽快忽慢的奔跑著，自己安裝後覺得很可愛所以在此分享給大家

專案名稱是 RunCat_for_windows
專案網址：https://github.com/Kyome22/RunCat_for_windows

他奔跑的快慢取決於當下 CPU 的使用率，不過偶爾會出現與工作管理員顯示不同調的情況（畢竟雙方都不是準確的測量工具）
另外也支援鸚鵡的造型！以及黑暗模式等等的功能。
安裝過程相當簡單，由 release 的地方下載最新版，解壓縮並執行 RunCat.exe 就可以啦

實際運行狀況如下：
![image](https://github.com/Kyome22/RunCat_for_windows/blob/master/RunCat/resources/runcat_demo.gif?raw=true)

雖不是什麼很有用的功能，但整天看著他跑心情也是相當不錯的

**Comment by JSHT** [comment]
這讓我想到這個專案 [PPCpuMon](https://github.com/tomozh/PPCpuMon#readme)
他一樣會在工具列中顯示，同時會根據 CPU 使用情況增加更多的 Party Parrot

**Comment by HsuChen88** [comment]
我的貓應該跑很慢..
差不多要走不動了

**Comment by yucheng1028** [comment]
太可愛了八

**Comment by Rosancl** [comment]
好可愛~~已下載

**Comment by cordydeng** [comment]
好有趣的小知識喔，想必原po一定是個愛貓的人

**Comment by henrywen547** [comment]
太可愛了吧
但這隻貓不能吸爆
阿不然會被當怪人

**Comment by jcpan821** [comment]
感謝分享

**Comment by d26488399** [comment]
可愛～感謝分享

**Comment by Lai27330** [comment]
謝謝分享，這樣以後趕著上線上課，結果電腦又跑不太動的時候，心情也不會這麼差了。

**Comment by yenwei12138** [comment]
可愛~有貓咪肯定給讚

**Comment by HaoFish123** [comment]
感謝分享

**Comment by treeann** [comment]
感謝分享!

**Comment by stan890425** [comment]
有貓貓就給讚!!

**Comment by sys511613** [comment]
很可愛欸 感謝分享

**Comment by jason79461385** [comment]
好酷 這讓我想到NZXT水冷頭上也有類似的東西呢

**Comment by alexthebigpp** [comment]
這種好可愛，但這樣反而增加分心的機會

**Comment by etin2468** [comment]
貓貓超可愛~~

**Comment by Lynn12378** [comment]
超可愛!感謝樓主分享

**Comment by RogelioKG** [comment]
好可愛😌
我記得 Android 11 也有一個養貓彩蛋：
https://3c.ltn.com.tw/news/41297
有水盆，食物，要定期補充給貓貓。
點按多次玩具可以吸引貓貓。
而且持續養的話，還會有越多隻貓貓跑來要你認養🥰

**Comment by brian431** [comment]
這還是我第一次看到這種東西，感覺還蠻有趣的，謝謝分享。能讓自己的工具列多一個特別的東西是件不錯的事情。
